llmfoundry/registry.py

```diff
@@ -339,6 +340,40 @@
     description=_config_transforms_description,
 )
 
+_load_planners_description = (
+    """The load_planners registry is used to register classes that implement the LoadPlanner interface.
+
+    The LoadPlanner will be passed as part of the FSDP config arg of the Trainer. It will be used to load distributed checkpoints.
+
+    Returns:
+        LoadPlanner: The load planner.
+    """
+)
+
+load_planners = create_registry(
+    'llmfoundry',
+    'load_planners',
+    generic_type=Type[LoadPlanner],
+    entry_points=True,
+    description=_load_planners_description,
+)
+
+_save_planners_description = (
+    """The save_planners registry is used to register classes that implement the SavePlanner interface.
+    The savePlanner will be passed as part of the FSDP config arg of the Trainer. It will be used to save distributed checkpoints.
```

Comment:
```suggestion
    The SavePlanner will be passed as part of the FSDP config arg of the Trainer. It will be used to save distributed checkpoints.
```